source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -36,290 +39,440 @@ The following table lists the upgrade paths from previous versions of the MinIO
+
+3. *(Optional)* To continue using Log Search API and Prometheus, add the following variables to the tenant's yaml specification file under ``.spec.env``
+
+   Use the following command to edit a tenant:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl edit tenants <TENANT-NAME> -n <TENANT-NAMESPACE>
+
+   - Replace ``<TENANT-NAME>`` with the name of the tenant to modify.
+   - Replace ``<TENANT-NAMESPACE>`` with the namespace of the tenant you are modifying.
+
+   Add the following values under ``.spec.env`` in the file:
+
+   .. code-block:: yaml
+      :class: copyable
+
+      - name: MINIO_LOG_QUERY_AUTH_TOKEN
+        valueFrom:
+          secretKeyRef:
+            key: MINIO_LOG_QUERY_AUTH_TOKEN
+            name: <TENANT_NAME>-log-secret
+      - name: MINIO_LOG_QUERY_URL
+        value: http://<TENANT_NAME>-log-search-api:8080
+      - name: MINIO_PROMETHEUS_JOB_ID
+        value: minio-job
+      - name: MINIO_PROMETHEUS_URL
+        value: http://<TENANT_NAME>-prometheus-hl-svc:9090
+
+   - Replace ``<TENANT_NAME>`` in the ``name`` or ``value`` lines with the name of your tenant.
+
+Upgrade Operator to |operator-version-stable|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. *(Optional)* Update each MinIO Tenant to the latest stable MinIO Version.
+ 
+   Upgrading MinIO regularly ensures your Tenants have the latest features and performance improvements.
+   Test upgrades in a lower environment such as a Dev or QA Tenant, before applying to your production Tenants.
+   See :ref:`minio-k8s-upgrade-minio-tenant` for a procedure on upgrading MinIO Tenants.
 
-      You can verify the currently installed Operator version by retrieving the object specification for an operator pod in the namespace.
-      The following example uses the ``jq`` tool to filter the necessary information from ``kubectl``:
+2. Verify the existing Operator installation.
+   Use ``kubectl get all -n minio-operator`` to verify the health and status of all Operator pods and services.
+   
+   If you installed the Operator to a custom namespace, specify that namespace as ``-n <NAMESPACE>``.
+
+   You can verify the currently installed Operator version by retrieving the object specification for an operator pod in the namespace.
+   The following example uses the ``jq`` tool to filter the necessary information from ``kubectl``:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get pod -l 'name=minio-operator' -n minio-operator -o json | jq '.items[0].spec.containers'
+   
+   The output resembles the following:
+   
+   .. code-block:: json
+      :emphasize-lines: 8-10
+
+      {
+         "env": [
+            {
+               "name": "CLUSTER_DOMAIN",
+               "value": "cluster.local"
+            }
+         ],
+         "image": "minio/operator:v4.5.8",
+         "imagePullPolicy": "IfNotPresent",
+         "name": "minio-operator"
+      }
+
+3. Download the Latest Stable Version of the MinIO Kubernetes Plugin
```

Comment:
This list item is title case and others are sentence case. Which is the usual style?

Reply:
We follow MongoDB. List items use sentence-style capitalization. Fixed this one.
And thanks for the good eye!